sdk/translation/azure-ai-documenttranslator/README.md

```diff
@@ -1,5 +1,10 @@
 # Azure Document Translator client library for Java
 
+## Deprecation Notice
+
+> **Note:** This SDK is deprecated and will be removed on March 2026. Please plan to migrate to use `azure-ai-translation-document`.
```

Comment:
```suggestion
> [!WARNING] 
> This SDK is deprecated and will be removed on March 2026. Please plan to migrate to use `azure-ai-translation-document`.
```

Reply:
The docs for md alert highlighting
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts